RecoTracker/LSTCore/src/alpaka/Kernels.h

```diff
@@ -220,15 +220,15 @@ namespace lst {
 
           for (unsigned int ix1 = 0; ix1 < nQuintuplets_lowmod1; ix1 += 1) {
             unsigned int ix = quintupletModuleIndices_lowmod1 + ix1;
-            if (quintupletsInGPU.partOfPT5[ix] || (quintupletsInGPU.isDup[ix] & 1))
+            if (quintupletsInGPU.isDup[ix] & 1)
               continue;
 
             for (unsigned int jx1 = 0; jx1 < nQuintuplets_lowmod2; jx1++) {
               unsigned int jx = quintupletModuleIndices_lowmod2 + jx1;
               if (ix == jx)
                 continue;
 
-              if (quintupletsInGPU.partOfPT5[jx] || (quintupletsInGPU.isDup[jx] & 1))
+              if (quintupletsInGPU.isDup[jx] & 1)
```

Comment:
Am I guessing roughly correctly that adding T5s from pT5s in `removeDupQuintupletsInGPUBeforeTC` at least doubles the loop size. (in this case the computation at least quadruples)

IIUC, `partOfPT5` both in ix and jx can be skipped: they can't add to dup cleanup. This should speed things up significantly.

Reply:
in the selection below 
I think that it's worth to check also the case with 4 overlapping hits, can be just `minNHitsForDup_T5 = 4` or if some downsides are visible with the same `minNHitsForDup_T5 = 5` but adding  `or (nMatched== minNHitsForDup_T5-1 and dR2 < 0.004)`